ui/app/mainui/activitycenter/views/ActivityNotificationBase.qml

```diff
@@ -31,8 +31,9 @@ Item {
         anchors.top: parent.top
         anchors.right: parent.right
         anchors.left: parent.left
-        messageTimestamp: notification.timestamp
-        previousMessageTimestamp: filteredIndex === 0 || !notification.previousTimestamp ? 0 : notification.previousTimestamp
+        messageTimestamp: notification ? notification.timestamp : 0
```

Comment:
Using Loader and sourceComponent i should check if `notification` exist because it's assigned after instantiation. I would be glad for solution without checks like this and without loader.setSource :(